From 345ccf93e9d00dfd0f71085832a7df0170b74f74 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Sun, 14 Jun 2026 17:42:19 +0200
Subject: [PATCH] src/intel/compiler/jay/jay_ir.h: do not used typed enum

This is a C23 feature and not supported by gcc < 13, which
many host distros still have (e.g. Debian 12).

Upstream-Status: Inappropriate [yocto host distro requirements]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
Upstream: N/A, code completely changed upstream
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 src/intel/compiler/jay/jay_ir.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/intel/compiler/jay/jay_ir.h b/src/intel/compiler/jay/jay_ir.h
index 8307044ef8a..abf175a71bf 100644
--- a/src/intel/compiler/jay/jay_ir.h
+++ b/src/intel/compiler/jay/jay_ir.h
@@ -579,7 +579,7 @@ jay_type_is_any_float(enum jay_type t)
    return jay_base_type(t) == JAY_TYPE_F || jay_base_type(t) == JAY_TYPE_BF;
 }
 
-enum jay_predication : uint8_t {
+enum jay_predication {
    /** No predication. */
    JAY_NOT_PREDICATED = 0,
 
@@ -634,7 +634,7 @@ typedef struct jay_inst {
    bool decrement_dep:1;
    unsigned padding  :12;
 
-   enum jay_predication predication;
+   uint8_t predication;
    enum jay_conditional_mod conditional_mod;
 
    jay_def cond_flag; /**< conditional flag */
-- 
2.55.0

